vcf2fhir/json_generator.py

```diff
@@ -97,6 +98,13 @@ def _add_region_studied(
         general_logger.debug("Region Uncallable %s", nocall_region[chrom])
         fhir_helper.add_regionstudied_obv(
             ref_seq, region_studied[chrom], nocall_region[chrom])
+    elif region_studied is not None and len(region_studied) == 0:
+        if conversion_region and not conversion_region[chrom].empty:
+            fhir_helper.add_regionstudied_obv(
+                    ref_seq,
+                    region_studied,
+                    nocall_region
+                )
```

Comment:
If think we can some how combine this if with above if on line 92. If you look at the code inside both the if and elif, they are same. That means you should be able to change the condition for first If such that it would call `add_regionstudied_obv` in both the case.

Reply:
> where did this logic go ? Was it not required ?

Actually the `Ranges Examined Component` was required at two locations:

`Line 55`

```
if (
                reportable_query_regions is not None and
                len(reportable_query_regions) == 0):
            obv_comp = observation.ObservationComponent()
            obv_comp.code = ranges_examined_component
```

and 

`Line 73`

```
    for _, row in reportable_query_regions.df.iterrows():
             obv_comp = observation.ObservationComponent()
             obv_comp.code = ranges_examined_component
```

so I created a variable named `ranges_examined_component` to avoid repetition.

```
ranges_examined_component = concept.CodeableConcept(
            {
                "coding": [
                    {
                        "system": "http://loinc.org",
                        "code": "51959-5",
                        "display": "Ranges-examined component"
                    }
                ]
            }
        )
```